ClientApp/src/components/Account/signupForm.js

```diff
@@ -0,0 +1,92 @@
+import React, { useContext, useState } from "react";
+import {
+  BoldLink,
+  BoxContainer,
+  FormContainer,
+  Input,
+  SubmitButton,
+  SmallText,
+} from "./common";
+import { Marginer } from "./marginer";
+import { LoginContext } from "./LoginContext";
+import { useForm } from "react-hook-form";
+
+export function SignupForm(props) {
+  
+  const { switchToSignin } = useContext(LoginContext);
+  const { register, handleSubmit } = useForm([]);
+  const { dublicateFound, setDublicateFound } = useState(false);
```

Comment:
useState returns array of size 2 and '[]' brackets are used for array destructuring, instead of '{}' which are meant for object destructuring in javascript.